### PR TITLE
fix: capitalization and small changes

### PR DESCRIPTION
**Pull request by strumswell**
See title.

### CODE REVIEW

File: content/DRAFT/index.html [hunk 1]
```diff
@@ -537,10 +537,9 @@ <h2>Digital Wallet Standards</h2>
       <section>
         <h2>Identifiers for Enterprise Identity</h2>
         <p>
-          A <a>DID</a> is a globally unique identifier developed specifically for decentralized systems as defined by
```
Comment (strumswell) [inline content/DRAFT/index.html:540]: I would not say that they are specifically developed for decentralized systems (even though it is literally in their name). Using `location-independent` is much more broad and is a better characterization for what they actually are. (W3C uses the same language btw https://www.w3.org/TR/did-core/#dids-as-enhanced-urns)

File: content/DRAFT/index.html [hunk 2]
```diff
@@ -570,11 +569,11 @@ <h2>Verifiable Data Registry & Endorsed <a>DID</a> Methods</h2>
           To be resolvable to <a>DID</a> documents, DIDs are typically recorded on an underlying system or network of
           some
           kind. Regardless of the specific technology used, any such system that facilitates the creation, verification,
-          updating, and/or deactivation of decentralized identifiers and <a>DID</a> documents is called a verifiable
```
Comment (strumswell) [inline content/DRAFT/index.html:573]: Why or?

File: content/DRAFT/index.html [hunk 3]
```diff
@@ -1010,9 +1009,8 @@ <h2><a>Credential</a> Revocation</h2>
         <p>
           It shall be understood that existing cached data can be used when the OCI Directory Service is not available
           for an update of the local cache due to a technical downtime. A <a>Digital Wallet</a> Provider SHALL store
-          communication request data when OCI Directory Service is not available to establish an audit trail and
-          document why revocation data older than 24 hours were potentially used for checking the revocation status over
-          the time period of the OCI Directory Service downtime.
+          communication request data when OCI Directory Service is not available to establish an audit trail. Revocation
+          data older than 24 hours SHALL be considered invalid and result in a failed credential verification.
```
Comment (strumswell) [inline content/DRAFT/index.html:1013]: Old revocation data is a security risk for the whole ecosystem. We need to give this info a max lifetime which, I propose, should not be any longer than 24h. The spec should be strict to force implementers finding a way to mitigate this risk as much as possible.

Comment (bluesteens) [reply content/DRAFT/index.html:1013]: P&A call Aug 10.
What happens if revocation list not available due to exceptional/catastrophic circumstances? We could tighten up the wording, but should leave room for using older cache. Consider adding control method requirement to check when revocation list is back online.
Ties into issue https://github.com/orgs/Open-Credentialing-Initiative/projects/2?pane=issue&itemId=21168468

**Pull request merged**